### PR TITLE
Add 404, move CSS files, change buttons, misc changes

### PR DESCRIPTION
**Pull request by SSS-Says-Snek**

### CODE REVIEW

**Comment by oddbookworm** [comment]
LGTM!

**Review by oddbookworm — APPROVED** [review]

**Comment by SSS-Says-Snek** [comment]
Ooh, also fixes #37

File: index.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 layout: default
+prev_page: [youtube.com, EEEE]
```
Comment (Matiiss) [inline index.html:3]: Umm, what's this?

File: index.html [hunk 2]
```diff
@@ -8,10 +9,10 @@ <h1>Introduction</h1>
     <p>
         "Where can I learn how to use pygame?" - one of the 
         most commonly asked questions that I get. But I always struggle to 
-        answer, I sometimes point them to the docs and examples, but that 
+        answer. I sometimes point them to the docs and examples, but that 
         isn't a comprehensive guide on building good pygame based applications.
```
Comment (Matiiss) [inline index.html:13]: Should _pygame based_ be hyphenated?

File: index.html [hunk 3]
```diff
@@ -8,10 +9,10 @@ <h1>Introduction</h1>
...
         There are many other tutorials and guides I found, but I always 
-        find something lacking, there are many comments we give and many nitpicks we have 
+        find something lacking.  There are many comments we give and many nitpicks we have
```
Comment (Matiiss) [inline index.html:15]: Past tense and present tense used together, should _found_ be changed to _find_ or _have found_ in which case the other _find_ changes to _found_? or _..., but there was always something lacking._ ?

File: index.html (hunk 1 above)
Comment (oddbookworm) [reply index.html:3]: the `prev_page` is something snek added in for buttons to the previous page and the next page.
I think the value is there as a placeholder (links to youtube, text on button is EEEE)

Comment (SSS-Says-Snek) [reply index.html:3]: Shoot I forgot to remove that

File: index.html (hunk 2 above)
Comment (SSS-Says-Snek) [reply index.html:13]: Maybe

File: index.html (hunk 3 above)
Comment (oddbookworm) [reply index.html:15]: Maybe this fix should be included with the first task in #63 since that's going to be editing the same content

Comment (SSS-Says-Snek) [reply index.html:15]: It might just be me, but I don't see anything wrong with the tense.

Comment (oddbookworm) [reply index.html:14]: Swapping between "I" and "we"

Comment (Matiiss) [reply index.html:14]: Alright, maybe just me then. ChatGPT said it couldn't find any issues either, so I guess this is resolved then and stays as is

**Review by Matiiss — APPROVED** [review]
LGTM

**Pull request merged**